File: src/main/java/aurocosh/divinefavor/common/state_mappers/InvisibleStateMapper.java
package aurocosh.divinefavor.common.state_mappers;


import net.minecraft.block.Block;
import net.minecraft.block.state.IBlockState;
import net.minecraft.client.renderer.block.model.ModelResourceLocation;
import net.minecraft.client.renderer.block.statemap.IStateMapper;

import java.util.Collections;
import java.util.Map;

public class InvisibleStateMapper implements IStateMapper {
    @Override
    public Map<IBlockState, ModelResourceLocation> putStateModelLocations(Block blockIn) {
        return Collections.emptyMap();
    }
}

File: src/main/java/aurocosh/divinefavor/common/state_mappers/InvisibleStateMapper.kt
package aurocosh.divinefavor.common.state_mappers


import net.minecraft.block.Block
import net.minecraft.block.state.IBlockState
import net.minecraft.client.renderer.block.model.ModelResourceLocation
import net.minecraft.client.renderer.block.statemap.IStateMapper

class InvisibleStateMapper : IStateMapper {
    override fun putStateModelLocations(blockIn: Block): Map<IBlockState, ModelResourceLocation> {
        return emptyMap()
    }
}
